runtime/runtime_darwin.go

```diff
@@ -0,0 +1,35 @@
+package runtime
+
+import (
+	"log"
+	"os/exec"
+)
+
+func (user *OSUser) Create(okIfExists bool) error {
```

Comment:
`ofIfExists` seems unused here. I'm guessing that's intentional, just wanted to make sure.

Reply:
Ah, I wanted to harmonise the function call across Windows and other platforms, so it could be called from platform-agnostic code. Currently user creation on darwin and linux is a TODO, so this code is currently uncalled.